src/pages/[platform]/build-a-backend/auth/auth-migration-guide/index.mdx

```diff
@@ -0,0 +1,436 @@
+```
+
+## Auth.deleteUserAttributes
+
+```diff
+-  import { Auth } from 'aws-amplify';
++  import { deleteUserAttributes } from 'aws-amplify/auth';
+
+-  Auth.deleteUserAttributes: (user: CognitoUser | any, attributeNames: string[]) => Promise<any>;
++  deleteUserAttributes: (input: DeleteUserAttributesInput) => Promise<void>;
+```
+
+## Auth.deleteUser
+
+```diff
+-  import { Auth } from 'aws-amplify';
++  import { deleteUser } from 'aws-amplify/auth';
+
+-  Auth.deleteUser: () => Promise<string | void>;
++  deleteUser: () => Promise<void>;
+```
+
+## Auth.updateUserAttributes
+
+```diff
+-  import { Auth } from 'aws-amplify';
++  import { updateUserAttributes } from 'aws-amplify/auth';
+
+-  Auth.updateUserAttributes: (user: CognitoUser | any, attributes: object, clientMetadata?: ClientMetaData) => Promise<string>;
++  updateUserAttributes: (input: UpdateUserAttributesInput) => Promise<UpdateUserAttributesOutput>;
+```
+
+## Auth.userAttributes
+
+```diff
+-  import { Auth } from 'aws-amplify';
++  import { fetchUserAttributes } from 'aws-amplify/auth';
+
+-  Auth.userAttributes: (user: CognitoUser | any) => Promise<CognitoUserAttribute[]>;
++  fetchUserAttributes: () => Promise<FetchUserAttributesOutput>;
+```
+
+## Auth.verifiedContact (DEPRECATED)
+
+This API has been deprecated: existing use cases can be migrated to the `fetchUserAttributes` API using `output.email_verified` and `output.phone_number_verified` attributes.
+
+```diff
+-  import { Auth } from 'aws-amplify';
++  import { fetchUserAttributes } from 'aws-amplify/auth';
+
+-  Auth.verifiedContact: (user: CognitoUser | any) => Promise<any>;
++  fetchUserAttributes: () => Promise<FetchUserAttributesOutput>;
+```
+
+## Auth.currentAuthenticatedUser (DEPRECATED)
+
+This API has been deprecated: existing use cases can be migrated to a combination of the `getCurrentUser` and `fetchAuthSession` API's.
+
+```diff
+-  import { Auth } from 'aws-amplify';
++  import { fetchAuthSession, getCurrentUser } from 'aws-amplify/auth';
+
+-  Auth.currentAuthenticatedUser: (params?: CurrentUserOpts) => Promise<CognitoUser | any>;
++  getCurrentUser: () => Promise<GetCurrentUserOutput>;
++  fetchAuthSession: (options?: FetchAuthSessionOptions) => Promise<AuthSession>;
+```
+
+## Auth.currentUserPoolUser (DEPRECATED)
+
+This API has been deprecated: existing use cases can be migrated to a combination of the `getCurrentUser` and `fetchAuthSession` API's.
+
+```diff
+-  import { Auth } from 'aws-amplify';
++  import { getCurrentUser } from 'aws-amplify/auth';
+
+-  Auth.currentUserPoolUser: (params?: CurrentUserOpts) => Promise<CognitoUser | any>;
++  getCurrentUser: () => Promise<GetCurrentUserOutput>;
+```
+
+## Auth.currentSession
+
+```diff
+-  import { Auth } from 'aws-amplify';
++  import { fetchAuthSession } from 'aws-amplify/auth';
+
+-  Auth.currentSession: () => Promise<CognitoUserSession>;
++  fetchAuthSession: (options?: FetchAuthSessionOptions) => Promise<AuthSession>;
+```
+
+
```

Comment:
There is an extra space between the `Auth.currentSession` and `Auth.userSession` sections.